charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 0.27.10
+version: 0.27.11
```

Comment:
this will need to be updated. as we merge PRs unfortunately this value keeps needing to be updated. due to the holidays i'm merging a lot of PRs and so there's a lot of contention on this version.

Reply:
I'm seeing `0.27.15` as the version in my fork as a result of this commit: https://github.com/grafana/helm-charts/commit/9efd3dec07a4bcb7bbc31ffc02c9523f67d6306b